pkg/placement/membership.go

```diff
@@ -264,7 +264,7 @@ func (p *Service) processRaftStateCommand(ctx context.Context) {
 			return
 
 		case <-processingTicker.C():
-			log.Debugf("Process Raft state command: nothing happened...")
+			// log.Debugf("Process Raft state command: nothing happened...")
```

Comment:
I agree with the change though I think this line should be deleted completely, or replaced with a proper code comment.

Reply:
Thinking about it, i don't think we even need the `processingTicker`. Its sole purpose was to show this log...

Reply:
Removed the ticker entirely